### ISSUE TITLE
Very few games have too fast music regardless of MHz setting

### ISSUE THREAD

**Issue by rsn8887** [MiSTer-devel/ao486_MiSTer#28]
Thank you for this amazing core that has kept me up late almost every night for two weeks testing and enjoying games like 4D Sports Driving, and other things like After Dark 4.0 and Johnny Castaway screensavers on Windows 95 👍 

Some games I tested like 
- Lode Runner Returns
- Warlords 2

play their fm music super fast, like 10 times too fast, regardless of MHz core setting.

I tried both Soundblaster and Adlib music settings, and OPL2 and OPL3, no change.

It is only those games out of 105 I tested, so no big deal.

**Comment by birdybro** [comment]
Have you already ran the setup utility to detect the soundcard again for Warlords 2 since updating the MiSTer core? Make sure the settings here are reflected.

```
SoundBlaster settings:

    address: 220h
    IRQ: 5
    DMA: 1
    type SoundBlaster 2.0

MIDI settings:

    address: 330h
    IRQ: 9

```

**Comment by boogermann** [comment]
There are many games that are very timing sensitive. Either sound will be all garbled, it will run too fast or scroll too fast. Warcraft 1, Indiana Jones and the Fate of Atlantis, Day of Tentacle, Dune,... are few of the ones I've tested.
Vogons has a nice [List of CPU speed sensitive games](https://www.vogonswiki.com/index.php/List_of_CPU_speed_sensitive_games)...

**Comment by rsn8887** [comment]
@birdybro : Yes and I double-checked those settings you mention.

@boogermann : I am pretty 100% sure it is not that. I tried different CPU speeds. By the way, Warcraft 1 and Dune are totally fine for me. As I mentioned, Lode Runner Returns and Warlords 2 are the problems.

**Comment by sorgelig** [comment]
both warlords 2 and lode runner use the same sound driver. 
I have no idea what timing source it uses. It seems doesn't use OPL timers and doesn't use PIT. 
If someone will give more details about this sound driver then i will think how to fix it.
Otherwise it will be left as is.

**Comment by rsn8887** [comment]
Fixed by release 20210107. Thank you!!!
